sbc_email/models/import_letters_history.py

```diff
@@ -281,17 +357,20 @@ def _manage_all_imported_files(self):
             listPaths = smb_conn.listPath(share_nas, imported_letter_path)
             for sharedFile in listPaths:
                 if func.check_file(sharedFile.filename) == 1:
-                    file_obj = BytesIO()
-                    smb_conn.retrieveFile(
-                        share_nas,
-                        imported_letter_path +
-                        sharedFile.filename,
-                        file_obj)
-                    file_obj.seek(0)
-                    self._copy_imported_to_done_letter(
-                        sharedFile.filename, file_obj, True)
+                    # when this is manual import we don't have to copy all
+                    # files, web letters are stock in the same folder...
+                    if not self.manual_import or self.is_in_list_letter(
```

Comment:
Since you already did the check `not self.manual_import` at line 302 (before calling the function), I guess it becomes useless here or not ?